Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -744,7 +746,10 @@ defineSuite([
                 expect(numberProcessing).toEqual(expected[1]);
             });
             viewRootOnly();
+            var spyUpdate = jasmine.createSpy('listener');
+            tileset.loadProgress.addEventListener(spyUpdate);
             scene.renderForSpecs();
+            expect(spyUpdate.calls.count()).toEqual(1);
             return Cesium3DTilesTester.waitForPendingRequests(scene, tileset);
         });
     });
```

Comment:
This test needs to be refactored more. The original `addEventListener` should be removed and the spy event listener should be checking all the same functionality. We are expecting 3 calls to be made, not 1. The reason it passed is because `scene.renderForSpecs` is only called once, so there's only 1 frame to do anything. Also you should be checking the values with `spyUpdate.calls.argsFor`.

What you will have to do here to cause all 3 events to trigger is check the spy when `waitingForPendingRequests` is finished. This promise is resolved when all the visible tiles are ready, and so all 3 events should be raised. You will want to do something roughly like:

```javascript
return Cesium3DTilesTester.waitForPendingRequests(scene, tileset) .then(function() {
    // Do spy checks here
});